kyaml/setters2/example_test.go

```diff
@@ -6,129 +6,9 @@ package setters2
 import (
 	"fmt"
 
-	"sigs.k8s.io/kustomize/kyaml/openapi"
 	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-// ExampleSet demonstrates using Set to replace the current field value in an object
```

Comment:
Why are we removing this? It seems like the use-case is still relevant with this change?

Reply:
This tests the cases of adding setters schema to global schema. Since we no longer do that with this change, removed it.